cypress/page/forms.page.ts

```diff
@@ -33,20 +33,20 @@ class PersonalFormPage {
       cy.visit(this.formURL);
     }
   
-    public fillForm(Info: PersonalInfo): void {
-      cy.get(this.firstName).type(Info.name);
-      cy.get(this.lastName).type(Info.lastName);
-      cy.get(this.email).type(Info.email);
-      cy.get(this.gender).find(`input[value = '${Info.gender}']`).click();
-      cy.get(this.mobile).type(Info.mobileNumber);
-      cy.get(this.birthDate).type(`{selectall}${Info.dateOfBirth}{enter}`);
-      Info.hobbies.forEach((hobbie) => {
+    public fillForm(info: PersonalInfo): void {
+      cy.get(this.firstName).type(info.name);
+      cy.get(this.lastName).type(info.lastName);
+      cy.get(this.email).type(info.email);
+      cy.get(this.gender).find(`input[value = '${info.gender}']`).click({force: true});
+      cy.get(this.mobile).type(info.mobileNumber);
+      cy.get(this.birthDate).type(`{selectall}${info.dateOfBirth}{enter}`);
+      info.hobbies.forEach((hobbie) => {
         cy.get(this.hobbies).find(".custom-control-label").filter(`:contains("${hobbie}")`).click();
```

Comment:
is the find needed? if so, you can try to modify the `this.hobbies` property to have this value `#hobbiesWrapper input+label`, in that case we are telling cypress to look in the labels

Reply:
The find is no neccesary . With this.hobbies is selecting the #hobbiesWrapper ID and then filter the hobbies.